### PR TITLE
Update theme description

### LINKED ISSUE #171: Add theme description
The theme needs a theme description in its `readme.txt`, and under the Description field in `style.css`. Currently it just uses a temporary "The WordPress default theme for 2022" message.

For example, here are the theme descriptions for the previous two default themes: 

**Twenty Twenty-One**

> Twenty Twenty-One is a blank canvas for your ideas and it makes the block editor your best brush.
> 
> With new block patterns, which allow you to create a beautiful layout in a matter of seconds, this theme’s soft colors and eye-catching — yet timeless — design will let your work shine.
> 
> Take it for a spin! See how Twenty Twenty-One elevates your portfolio, business website, or personal blog.

**Twenty Twenty**

> Our default theme for 2020 is designed to take full advantage of the
flexibility of the block editor. Organizations and businesses have the
ability to create dynamic landing pages with endless layouts using the
group and column blocks. The centered content column and fine-tuned
typography also makes it perfect for traditional blogs. Complete editor
styles give you a good idea of what your content will look like, even
before you publish. You can give your site a personal touch by changing
the background colors and the accent color in the Customizer. The colors of
all elements on your site are automatically calculated based on the colors
you pick, ensuring a high, accessible color contrast for your visitors.

### PR DESCRIPTION
**Pull request by kjellr**
Closes https://github.com/WordPress/twentytwentytwo/issues/171. 

One minor change: I went with "subtle styles" instead of just "styles" [as I'd mentioned here](https://github.com/WordPress/twentytwentytwo/issues/171#issuecomment-970526922). I'd hoped to work that word in somewhere, since the styles are purposefully subtle. This seemed like a good place for it.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -9,7 +9,11 @@ License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
 == Description ==
 
-The WordPress default theme for 2022.
+Built on a solidly designed foundation, Twenty Twenty-Two embraces the idea that everyone deserves a truly unique website. The theme’s subtle styles are inspired by the diversity and versatility of birds: its typography is lightweight yet strong, its color palette is drawn from nature, and its layout elements sit gently on the page.
+
+The true richness of Twenty Twenty-Two lies in its opportunity for customization. The theme is built to take advantage of the Full Site Editing features built into WordPress 5.9, which means that colors, typography, and the layout of every single page on your site can be customized to suit your vision. It also includes dozens of block patterns, opening the door to a wide range of professionally designed layouts in just a few clicks.
```
Comment (jffng) [inline readme.txt]:
```suggestion
The true richness of Twenty Twenty-Two lies in its opportunity for customization. The theme is built to take advantage of the Full Site Editing features introduced in WordPress 5.9, which means that colors, typography, and the layout of every single page on your site can be customized to suit your vision. It also includes dozens of block patterns, opening the door to a wide range of professionally designed layouts in just a few clicks.
```

**Review by jffng — APPROVED** [review]
I left one small suggestion, either way LGTM.

File: readme.txt (hunk 1 above)
Comment (kjellr) [reply readme.txt]: Good suggestion. I'll change that in `style.css` too and then merge. 👍

File: style.css [hunk 2]
```diff
@@ -3,7 +3,7 @@ Theme Name: Twenty Twenty-Two
 Theme URI: https://github.com/wordpress/twentytwentytwo/
 Author: the WordPress team
 Author URI: https://wordpress.org/
-Description: The WordPress default theme for 2022.
+Description: Built on a solidly designed foundation, Twenty Twenty-Two embraces the idea that everyone deserves a truly unique website. The theme’s subtle styles are inspired by the diversity and versatility of birds: its typography is lightweight yet strong, its color palette is drawn from nature, and its layout elements sit gently on the page. The true richness of Twenty Twenty-Two lies in its opportunity for customization. The theme is built to take advantage of the Full Site Editing features built into WordPress 5.9, which means that colors, typography, and the layout of every single page on your site can be customized to suit your vision. It also includes dozens of block patterns, opening the door to a wide range of professionally designed layouts in just a few clicks. Whether you’re building a single-page website, a blog, a business website, or a portfolio, Twenty Twenty-Two will help you create a site that is uniquely yours.
```
Comment (kjellr) [inline style.css]:
```suggestion
Description: Built on a solidly designed foundation, Twenty Twenty-Two embraces the idea that everyone deserves a truly unique website. The theme’s subtle styles are inspired by the diversity and versatility of birds: its typography is lightweight yet strong, its color palette is drawn from nature, and its layout elements sit gently on the page. The true richness of Twenty Twenty-Two lies in its opportunity for customization. The theme is built to take advantage of the Full Site Editing features introduced in WordPress 5.9, which means that colors, typography, and the layout of every single page on your site can be customized to suit your vision. It also includes dozens of block patterns, opening the door to a wide range of professionally designed layouts in just a few clicks. Whether you’re building a single-page website, a blog, a business website, or a portfolio, Twenty Twenty-Two will help you create a site that is uniquely yours.
```

**Pull request merged**